### PR TITLE
hostname can be specified with a trailing '/' without error

### LINKED ISSUE #9: Trailing '/' on host addr leads to failure
When running
```
lgn = Lightning.(host=hostname)
```
a training '/' on the hostname will cause an error when later creating a session and the error message is not overly helpful.

### PR DESCRIPTION
**Pull request by jwittenbach**
Fixes #9

### CODE REVIEW

**Comment by jwittenbach** [comment]
My journey to learn/use/maybe-contribue-to Lightning begins today...starting small =)

File: lightning/main.py [hunk 1]
```diff
@@ -5,6 +5,9 @@
 class Lightning(object):
 
     def __init__(self, host="http://localhost:3000", ipython=False, auth=None):
+        if host[-1] == '/':
+            host = host[:-1]
+
         self.host = host
```
Comment (mathisonian) [inline lightning/main.py]: Could you just change this to 

```
self.set_host(host)
```

to avoid duplicating the logic?

**Comment by mathisonian** [comment]
Heck yes!

Would you mind making the small update I commented on and we'll pull this in?

File: lightning/main.py (hunk 1 above)
Comment (jwittenbach) [inline lightning/main.py]: Good call!

**Pull request merged**